definitions/ext-rf_scanner/dashboard.json

```diff
@@ -0,0 +1,173 @@
+{
+  "name": "RF Device",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "entity dashboard",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 3,
+            "width": 4
+          },
+          "title": "Details by Device",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "from THD_RF_TRANSACTION select  latest(serialNumber),latest(ipAddress),latest(lastActiveTimestamp) since 12 hours ago"
```

Comment:
i'm not aware if those since 12 hours ago will make the time picker not work. but we can try out